src/components/ScrollView.re

```diff
@@ -137,7 +137,9 @@ external make:
     ~shouldRasterizeIOS: bool=?,
     ~style: Style.t=?,
     ~testID: string=?,
-    ~children: React.element=?
+    ~children: React.element=?,
+    // temporary react-native-web-prop
+    ~useWindowScrolling: bool=?
```

Comment:
This thing doesn't seems to be in RNW repo. Can you move this back & keep it in a patch if you use a specific hacked RNW version? :D

Reply:
I have removed it, be sure to add it in your personal patch!

Reply:
Sorry forget this PR was on my master branch! This was a personal patch for a PR that solves a current issue in react-native-web with window scrolling.

Normally I create a branch specifically for PR's will do again for the next PR :D